src/Zitadel/Extensions/ClaimsPrincipalExtensions.cs

```diff
@@ -15,7 +15,14 @@ public static class ClaimsPrincipalExtensions
     /// <param name="roles">List of roles. One of them must be present on the principal.</param>
     /// <returns>True if any role is on the principal. False otherwise.</returns>
     public static bool IsInRole(this ClaimsPrincipal principal, params string[] roles) =>
-        roles.Select(principal.IsInRole).Any();
+        roles.Any(principal.IsInRole);
+
+    public static bool IsInRole2(this ClaimsPrincipal principal, params string[] roles)
```

Comment:
IsInRole2 ? This is demo / showcase code right?

Reply:
Right.